docs/views/about.html

```diff
@@ -0,0 +1,67 @@
+{% extends "layout.html" %}
+
+{% block page_title %}
+About - GOV.UK Prototype kit
+{% endblock %}
+
+{% block content %}
+
+<main id="content" role="main">
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <div class="breadcrumbs">
+        <ol>
+          <li><a href="/docs">GOV.UK Prototype kit</a></li>
+        </ol>
+      </div>
+      <h1 class="heading-xlarge">About</h1>
+
+      <p class="lede">
```

Comment:
I can add them below the current content on the homepage, but I'm not sure that will help them be seen - I imagine most will follow one of the main links.